perf/perf_c2c.py

```diff
@@ -92,23 +80,30 @@ def test_c2c(self):
         # When input is used for report, then need to pass the file argument
         # to the same file, record should log the data. So altering record,
         # report options to have the proper arguments.
+        output_file = "perf.data"
         if self.report == "-i":
-            self.report = "-i %s" % self.temp_file
-            self.record = self.record + " -o %s" % self.temp_file
+            self.report = "-i %s" % output_file
         elif self.report == "--input":
-            self.report = "--input=%s" % self.temp_file
-            self.record = self.record + " -o %s" % self.temp_file
+            self.report = "--input=%s" % output_file
+        elif self.report in ['-k', '--vmlinux']:
```

Comment:
The commit message states that absolute path to kernel binary is required starting with RHEL 9.1. Is this true for other releases as well. The above change unconditionally applies to all releases. Will this change work for older releases?

Reply:
With earlier versions -k & --vmlinux works with any provided input which is fixed in rhel 9.1. Tried with this change in SLES15 SP4 and RHEL 8.7 works fine.

Reply:
okay thanks for the confirmation.